source/LibUISharp/src/LibUISharp/SafeHandles/SafeControlHandle.cs

```diff
@@ -21,6 +21,7 @@ public sealed class SafeControlHandle : SafeHandleZeroIsInvalid
         /// Returns the native control handle for this control.
         /// </summary>
         /// <returns>An <see cref="UIntPtr"/> representing the controls handle.</returns>
+        [CLSCompliant(false)]
         public UIntPtr DangerousGetControlHandle() => LibuiLibrary.uiControlHandle(handle);
```

Comment:
This actually isn't used anywhere in the code, which obviously doesn't mean there isn't a use case for it, but it can be safely removed for now.